Form_Checkout.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Checkout</title>
+    <script src="https://kit.fontawesome.com/e4353a2c7e.js" crossorigin="anonymous"></script>
+   <link rel="stylesheet" href="formstyle.css">
+</head>
+
+<body>
+    <div class="row">
```

Comment:
Please consider using `HTML5` semantic tags. For example:
```suggestion
    <main class="row">
```